nuxeo-core/nuxeo-core-management/src/main/java/org/nuxeo/ecm/core/management/probes/ProbeManagerImpl.java

```diff
@@ -44,10 +48,14 @@
 
     protected final Map<String, Probe> probesByShortcuts = new HashMap<String, Probe>();
 
+    protected final Map<String, ProbeInfo> probesForHealthCheck = new HashMap<String, ProbeInfo>();
+
     protected final Set<ProbeInfo> failed = new HashSet<ProbeInfo>();
 
     protected final Set<ProbeInfo> succeed = new HashSet<ProbeInfo>();
 
+    protected final int healthCheckIntervalInSec = 20;
```

Comment:
Define `public static final int DEFAULT_HEALTH_CHECK_INTERVAL_SECONDS = 20;` and use it instead of hardcoded `20`. It's good to keep a field `healthCheckIntervalInSec` as we'll probably want to make the value configurable through a configuration property.